.bcr/api/metadata.template.json

```diff
@@ -0,0 +1,22 @@
+{
+    "homepage": "https://www.envoyproxy.io/",
+    "maintainers": [
+        {
+            "github": "mering",
```

Comment:
@mering ill land this for now but wanted to confirm you are happy being listed as a bcr maintainer

we have an envoy-bzlmod channel on the envoy slack - would you be able to jump on there if so?